### DISCUSSION TITLE
Question about the current MQTT entries in the changelog

### DISCUSSION THREAD

**Discussion by torbennehmer** [fredlcore/BSB-LAN#354] [Q&A] [answered]
Hi @fredlcore, @dukess or @Luposoft63 

I've a question about the current changelog:

> New MQTT functions, including running BSB-LAN as a MQTT broker

I was trying to find the part of the code, that made BSB-LAN a MQTT broker, but I am unable to do so. Could you give me a quick hint here, where to start looking?

Best regards & thanks!
Torben

**Comment by fredlcore** [comment, marked as answer]
Hm, now that you mention it, I may have got this wrong, but I understood the latest additions by @Luposoft63 to provide a broker-like function, for example for setting a parameter.

**Reply by torbennehmer** [reply]
Hm, I think that the terms got intermixed then. The "broker" is the MQTT server part, the piece of software that manages the messages, keeps track of subscriptions etc.

What we did get was a first implementation to set any value or to initiate a query by message. So if @Luposoft63 would agree, i'd put something like this in the changelog:

- Allow any parameter to be set by an MQTT message
- Actively query any parameter once by sending an MQTT message

**Reply by fredlcore** [reply]
Thanks for the clarification, will change it accordingly.
